tests/wpt/web-platform-tests/html/semantics/forms/form-submission-0/submit-entity-body.html

```diff
@@ -5,14 +5,52 @@
 <script>
 var simple_tests = [
   {
-    name: "form submission should navigate to url with x-www-form-urlencoded",
+    name: "form submission from form should navigate to url with x-www-form-urlencoded",
     input: "<input name=foo value=bara>",
-    enctype: "application/x-www-form-urlencoded"
+    enctype: "application/x-www-form-urlencoded",
+    submitelement: "",
+    elementid: "testform",
```

Comment:
You might wanna revert this. I don't think this is DRYer than what you previously had.